### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by feng-tao**
Add google group info.

### CODE REVIEW

**Comment by feng-tao** [comment]
cc @ttannis

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Please visit [Roadmap](docs/roadmap.md) if you are interested in Amundsen upcomi
 
 ## Community
 
-Want help? Join our [slack channel](https://bit.ly/2tFsN1X)
+Want help? Join our [slack channel](https://bit.ly/2tFsN1X). For other discussion, please use the [Amundsen mailing list](amundsen-dev@googlegroups.com).
```
Comment (ttannis) [inline README.md]: We need to use email markup for this or else it treats it as normal link and goes to a 404 page. 

Also does this alias with `@googlegroups.com` work, can you send a test email?  To my knowledge I thought all we had setup was `amundsen-dev@lyft.com`.

Comment (feng-tao) [reply README.md]: this is not our email alias. I manually created this google group for external people. Currently I just add myself which requires others to join the group by themselves.

**Comment by feng-tao** [comment]
cc @danwom

**Pull request closed without merging**